common/web/input-processor/package.json

```diff
@@ -35,7 +35,7 @@
     "@keymanapp/keyboard-processor": "*",
     "@keymanapp/lexical-model-layer": "*",
     "@keymanapp/models-types": "*",
-    "@keymanapp/web-environment": "*",
+    "@keymanapp/keyman-version": "*",
```

Comment:
given this, we'll probably need to build keyman-version instead in build.sh for input-processor